docs/setup_eecs281.md

```diff
@@ -82,6 +80,29 @@ If you've used Visual Studio on your computer before, start the tutorial at the
 
 If Visual Studio is new to you or new to your computer, start at the [beginning](https://eecs280staff.github.io/p1-stats/setup_visualstudio.html).  Stop after you've completed the [Add existing files](https://eecs280staff.github.io/p1-stats/setup_visualstudio.html#add-existing-files) section.
 
+#### `getopt` library
+Visual Studio does not ship with a `getopt` library, which processes command line arguments.  We'll put a copy in our project directory.
+
+Download [`wingetopt.h`](wingetopt.h) and place it in your project source code directory.
+
+Include the library in your program.  Notice that we use the local copy only on windows.  The Autograder will use its preinstalled version of `getopt`.
+```c++
+#ifdef _WIN32
```

Comment:
Give this version a try.  I made a header-only version.  We can always revert commit 4daa14ab1405382bb49458c456298974597cd8d4 if this doesn't work out.

Reply:
See https://github.com/eecs280staff/tutorials/pull/40#discussion_r1037639084.